src/eos-updater-poll-common.c

```diff
@@ -574,6 +583,12 @@ parse_latest_commit (OstreeRepo           *repo,
   else
     *out_redirect_followed = FALSE;
 
+  version = g_variant_lookup_value (metadata, "version", G_VARIANT_TYPE_STRING);
```

Comment:
Might be nice to use the `OSTREE_COMMIT_META_KEY_VERSION` macro even though `version` is a well known key.